### DISCUSSION TITLE
#44: Can't reach 100%, don't know why 😭

### DISCUSSION THREAD

**Discussion by thoughtlessnerd** [cssbattle/forum#43] [General]
![image](https://user-images.githubusercontent.com/79973672/113538466-f3d23600-95f8-11eb-9208-31c2c3c0a5c8.png)
Please help. its at that point where i cant even see the difference with naked eye.

**Comment by SelenIT** [comment]
For some reason, that target turns out really hard to solve using `border-radius` since the latest backend update. Haven’t you tried other ways to create circles? Maybe [this thread can help](https://github.com/cssbattle/forum/discussions/12#discussioncomment-452535).

**Reply by thoughtlessnerd** [reply]
thanks for the solution ☺
